ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-stable.yaml

```diff
@@ -532,6 +519,7 @@ tests:
       ADDITIONAL_WORKER_VM_TYPE: t2a-standard-4
       ADDITIONAL_WORKERS: "1"
       COMPUTE_NODE_REPLICAS: "2"
+      ENABLE_OTA_TEST: OCP-60396
```

Comment:
Maybe you can choose another similar job to enable OCP-60396 or just update `aws-ipi-ovn-ipsec-to-multiarch-f14` to `aws-ipi-ovn-ipsec-f14`

Reply:
can you explain?

Reply:
Because you deleted `aws-ipi-ovn-ipsec-to-multiarch-f14` and enabled `OCP-60396` in other job, that means there will no job covers `aws-ipi-ovn-ipsec`, the PR changed test coverage.

Reply:
I see. I haven't deleted `aws-ipi-ovn-ipsec-to-multiarch` but moved it from x86 to arm (where it used to be few versions ago). I need it in arm64 cause currently we have no coverage whatsoever for arm-to-multi. my thought is if we have this profile under arm64 we don't need it under amd64 as well, but if you think otherwise, I can just as well leave both :thinking:

Reply:
I am not sure if it is reasonable, @lwan-wanglin can you take a look if we can move aws-ipi-ovn-ipsec-to-multiarch from amd64 to arm64?

Reply:
Personally I think it is a good idea to move the profile from amd to arm for a boarder coverage, also saved our cloud cost.

Reply:
in 4.14, this profile is located in arm64 https://github.com/openshift/release/blob/e413b5f9f39fc78bf320030bdb609e247eca09da/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-nightly.yaml#L325
however, now that we speak, I checked and seen it has been moved to https://github.com/openshift/release/blob/e413b5f9f39fc78bf320030bdb609e247eca09da/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__arm64-nightly.yaml#L298-L317
where it does not use stable and not `openshift-upgrade-qe-test-arch-migration` but rather `openshift-upgrade-qe-test-heterogeneous` so not suitable for my tests. however since as it turns out it does have the profile on arm, now I'm open to suggestions what is the best move in terms of coverage. one thing for sure, I need at least one arm64-to-multiarch profile that uses `openshift-upgrade-qe-test-arch-migration` on a stable build. nightly would not work.

Reply:
Ok, sounds good. Thanks.

Reply:
updated